src/app.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  render() {
+    return (
+      <div>
+        Charity data to be rendered dynamically here!
+      </div>
+    )
+  }
+}
+
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
I think this whole file is probably not needed. The way this reads is if it's going to be duplicating the work you're already doing in your charities repo. I believe you've got this file's work covered by line 7 of your index.html.